src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,691 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="VkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">500</property>
+    <property name="default-height">550</property>
+    <property name="modal">True</property>
+    <property name="title" translatable="yes" comments="vkBasalt is a Vulkan post processing layer for Linux">vkBasalt Settings</property>
+    <child>
+      <object class="GtkShortcutController">
+        <child>
+          <object class="GtkShortcut">
+            <property name="trigger">Escape</property>
+            <property name="action">action(window.close)</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <property name="show-start-title-buttons">False</property>
+            <property name="show-end-title-buttons">False</property>
+            <child type="start">
+              <object class="GtkBox">
```

Comment:
you don't need a container here, just set the button as a direct child of the headerbar.

Reply:
Like this? https://github.com/bottlesdevs/Bottles/pull/1668/commits/522e3ccf733662c0c95dc6ba2fe128f1a6e876db

Reply:
Yes. You can also use `type=start` by the way, but it was never needed in the first place, all children of a headerbar are added to the start of the widget unless `=end` is specified.